README.md

```diff
@@ -22,12 +22,30 @@ urlFragment: uwp-signing-in-graph-aad
 
 ## About this sample
 
+### Table of content
+- [About this sample](#about-this-sample)
+  - [Table of content](#table-of-content)

```

Comment:
```suggestion
  - [Table of contents](#table-of-contents)

```

Reply:
Removed from TOC as it's not required.
Thanks!